docs/dev/build-instructions/faq.md

```diff
@@ -20,3 +20,35 @@ If for instance you have installed OpenSSL to `/usr/local/opt/openssl`, you can
 cmake -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl ..
 ```
 
+
+### Build a specific version or commit
+
+#### Version
+
+Use tags to build a specific release version of Mumble.
+You can find the `tag`-list [here](https://github.com/mumble-voip/mumble/tags) or by using the command `git tag` inside a cloned directory.
+
+Use:
+
+`git clone -b [tag] https://github.com/mumble-voip/mumble.git`
```

Comment:
````suggestion
```
git clone -b [tag] https://github.com/mumble-voip/mumble.git
```
````

Reply:
Ok, I was thinking about this, what do you think?:
```

```bash

```
?

Reply:
Using `bash` as the highlighting language is fine to me. Though I think it doesn't influence the outcome (on GitHub).

```
git clone -b [tag] https://github.com/mumble-voip/mumble.git
```
vs (now with bash tag)
```bash
git clone -b [tag] https://github.com/mumble-voip/mumble.git
```

Reply:
Oh...I thought it would do something. Well in that case, I will simply implement your changes.